docs/runtime/training_cnn_on_mnist.md

```diff
@@ -0,0 +1,107 @@
+# Training a simple CNN model on MNIST using ONERT
+
+Let's train a simple neural network on MNIST dataset using ONERT.
+
+## Prepare dataset
+
+MNIST dataset consists of 60,000 training images and 10,000 test images. Each image is a 28x28 grayscale image, associated with a label from 10 classes. This dataset is available in TensorFlow datasets. ONERT provides a `tf_dataset_converter` tool to download dataset from TensorFlow Datasets and converts it to binary format for ONERT training.
+
+```bash
+$ python3 tools/generate_datafile/tf_dataset_converter/main.py \
+--dataset-name fashion_mnist --prefix-name fashion-mnist --model mnist
+Shape of images : (28, 28, 1)
+Shape of labels: () <dtype: 'int64'>
+class names[:10]  : ['T-shirt/top', 'Trouser', 'Pullover', 'Dress', 'Coat', 'Sandal', 'Shirt', 'Sneaker', 'Bag', 'Ankle boot']
+class length      : 10
+The data files are created!
+$ tree out
+out
+├── fashion-mnist.test.input.100.bin
+├── fashion-mnist.test.output.100.bin
+├── fashion-mnist.train.input.1000.bin
+└── fashion-mnist.train.output.1000.bin
+
+0 directories, 4 files
+```
+
+## Prepare model
+
+Prepare a simple neural network for trainng MNIST datasets using the `tf.keras` API. This code generates a `model.tflite` TFLite model file.
+
+```python
+import tensorflow as tf
+import tensorflow_datasets as tfds
+
+# Create a model
+model = tf.keras.models.Sequential([
+    tf.keras.layers.Flatten(input_shape=(28, 28)),
+    tf.keras.layers.Dense(128, activation='relu'),
+    tf.keras.layers.Dense(10)
+])
+
+# Donwload train dataset
+ds = tfds.load('mnist', split='train', as_supervised=True)
+ds = ds.batch(128)
+
+model.compile(optimizer='adam', loss='mean_squared_error')
+model.fit(ds, epochs=5)
+
+converter = tf.lite.TFLiteConverter.from_keras_model(model)
+tflite_model = converter.convert()
+
+with open('model.tflite', 'wb') as f:
+    f.write(tflite_model)
+```
+
+## Convert tflite model to circle model
+
+ONERT takes both the tflight model and the circle model. So you can use this tflight model file directly. To use the circle model instead of the tfleet model, you must convert it to the circle model using the 'onecc' tool, and 'onecc' can only be used in Ubuntu. When the ‘one-compiler’ debian package is installed on the ONE release page, it is installed in the ‘/usr/share/one/bin’ directory. Please refer to the help for detailed guidelines. The following command converts the tflight model into a circle model.
```

Comment:
```suggestion
ONERT takes both the tflight model and the circle model. So you can use this tflight model file directly. To use the circle model instead of the tflitemodel, you must convert it to the circle model using the 'onecc' tool, and 'onecc' can only be used in Ubuntu. When the ‘one-compiler’ debian package is installed on the ONE release page, it is installed in the ‘/usr/share/one/bin’ directory. Please refer to the help for detailed guidelines. The following command converts the tflight model into a circle model.
```